README.adoc

```diff
@@ -36,30 +38,238 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 
 [[initial]]
-== Create a resource controller
+== Add required dependencies
+
+To use the Spring Integration GCP Pub/Sub channel adapters, you'll need to pull in the
+`spring-integration-gcp` dependency. We also strongly recommend the Spring Cloud GCP Pub/Sub Boot
+starter, `spring-cloud-gcp-starters-pubsub`, to deal with all the object instantiation and auto
+configuration.
+
+At the time of writing, the Spring Cloud GCP libraries are still in Alpha stage. As such, they are
+only published to Spring Snapshots Maven repository.
+
+To pull in these dependencies, add the following to your `pom.xml` file if you're using Maven:
+
+```
+<dependencies>
+    ...
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-gcp-starter-pubsub</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-integration-gcp</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </dependency>
+    ...
+</dependencies>
+
+
+
+<repositories>
+    ...
+    <repository>
+        <id>repository.spring.snapshot</id>
+        <name>Spring Snapshot Repository</name>
+        <url>http://repo.spring.io/snapshot</url>
+    </repository>
+    ...
+</repositories>
+```
+
+Or, if you're using Gradle:
+
+```
+repositories {
+    ...
+    maven {
+        url "http://repo.spring.io/snapshot"
+    }
+    ...
+}
+
+dependencies {
+    ...
+    compile("org.springframework.cloud:spring-cloud-gcp-starter-pubsub:1.0.0.BUILD-SNAPSHOT")
+    compile("org.springframework.cloud:spring-integration-gcp:1.0.0.BUILD-SNAPSHOT")
+    ...
+}
+```
+
+== Create application files
```

Comment:
You're supposed to go from the `initial` folder to the `complete` folder. The application files need to be created before content is added to them below, piece by piece.